tests/store/crud/read/test_read_case.py

```diff
@@ -50,40 +50,40 @@ def test_get_cases_by_customers_action_and_case_search_pattern(
         assert case_search in case.name
 
 
-def test_get_cases_by_customer_pipeline_and_case_search_pattern(
+def test_get_cases_by_customer_workflow_and_case_search_pattern(
     store_with_cases_and_customers: Store,
 ):
     """Test that only cases with the specified customer, pipeline, and case search pattern are returned."""
     # GIVEN a store with some cases and customers
     case = store_with_cases_and_customers._get_query(table=Case).first()
 
-    # Set the pipeline and case_search
+    # Set the workflow and case_search
     customer = case.customer
     pipeline = case.data_analysis
```

Comment:
or maybe analysis is more suitable?
```suggestion
    workflow = case.data_analysis
```

Reply:
Done